### PR TITLE
8th deprecate stow_path, update readme for commitment

### PR DESCRIPTION
**Pull request by red1408**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,8 +144,7 @@ spec:
             - "/import/bin/import"
             - "--dimse_aet=IMPORTADAPTER"
             - "--dimse_port=2575"
-            - "--dicomweb_addr=https://healthcare.googleapis.com/v1beta1"
-            - "--dicomweb_stow_path=/projects/myproject/locations/us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb/studies"
+            - "--dicomweb_address=https://healthcare.googleapis.com/v1beta1/projects/myproject/locations/us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb"
 ```
 
 If needed, to additionally include an Export Adapter, you can add the to the
```
Comment (danielbeaudreau) [inline README.md]: The export adapter also has a split peer dicomweb addr and peer dicomweb path. I'm thinking about deprecating that too, although for this CL I think it's fine.

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java [hunk 2]
```diff
@@ -19,46 +19,55 @@
...
+      description =
+          "Path to send StowRS requests for DicomWeb peer. This is appended to the contents of --dicomweb_addr flag."
   )
   String dicomwebStowPath = "";
 
   @Parameter(
-    names = {"--oauth_scopes"},
-    description = "Comma seperated OAuth scopes used by adapter."
+      names = {"--dicomweb_address"},
+      description = "Address for DicomWeb service. Must be a full path (including /dicomWeb in case of healthcare api)."
```
Comment (danielbeaudreau) [inline import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: Edit to phrasing at the end: "Must be a full path up to /dicomWeb if the Cloud Healthcare API is used."

File: README.md [hunk 3]
```diff
@@ -10,9 +10,10 @@ The Import Adapter converts incoming DIMSE requests to corresponding DICOMWeb re
 - C-STORE to STOW-RS
 - C-FIND to QIDO-RS
 - C-MOVE uses QIDO-RS to determine which instances to transfer, then for each instance executes a 
-WADO-RS request to fetch the instance and a C-STORE request to transfer it to the C-MOVE destination 
+WADO-RS request to fetch the instance and a C-STORE request to transfer it to the C-MOVE destination
+- Storage commitment service to QIDO-RS
 
-Available AET destinations for C-MOVE are configured via an AET dictionary json file, 
+Available AET destinations for C-MOVE and storage commitment service are configured via an AET dictionary json file,
```
[the remark this answers is not in the document]
Comment (red1408) [reply README.md]: Ok

File: README.md [hunk 4]
```diff
@@ -50,10 +51,18 @@ And command to create configmap from it:
 kubectl create configmap aet-dictionary --from-file=AETs.json
 ```
 
+AET dictionary json can also passed directly via "--aet_dictionary_inline" parameter.
+
 Note that any C-FIND query on the ModalitiesInStudy tag will result in 1 QIDO-RS query per modality.
 
 For the list of command line flags, see [here](import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java)
 
+### Import Adapter parameter deprecation notes
```
[the remark this answers is not in the document]
Comment (red1408) [reply README.md]: Ok

File: README.md [hunk 5]
```diff
@@ -50,10 +51,18 @@ And command to create configmap from it:
 kubectl create configmap aet-dictionary --from-file=AETs.json
 ```
 
+AET dictionary json can also passed directly via "--aet_dictionary_inline" parameter.
```
[the remark this answers is not in the document]
Comment (red1408) [reply README.md]: Ok

File: README.md (hunk 1 above)
Comment (red1408) [reply README.md]: For StowRsSender (sink) the parameters are called peer_dicomweb_addr and peer_dicomweb_stow_path, these can be deprecated in the same way. 
But DicomClient that is used to for wado-rs (source) gets --dicomweb_addr from config (that goes only up to api version) and the rest from incoming Pubsub message. I guess this is where addr + stow path convention originated from and where it actually makes sense.

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java (hunk 2 above)
Comment (red1408) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: Ok

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java [hunk 6]
```diff
@@ -19,46 +19,57 @@
...
+      description = "Port the server is listening to for incoming DIMSE requests."
   )
   Integer dimsePort = 0;
 
+  @Deprecated
   @Parameter(
-    names = {"--dicomweb_addr"},
-    description = "Address for DicomWeb service."
+      names = {"--dicomweb_addr"},
+      description = "Address for DicomWeb service. Deprecated and used only with C-STORE."
```
Comment (danielbeaudreau) [inline import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: Missing a space here.

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java [hunk 7]
```diff
@@ -19,46 +19,57 @@
...
   String dicomwebAddr = "";
 
+  @Deprecated
   @Parameter(
-    names = {"--dicomweb_stow_path"},
-    description =
-        "Path to send StowRS requests for DicomWeb peer. This is appended to the contents of --dicomweb_addr flag."
+      names = {"--dicomweb_stow_path"},
+      description =
+          "Path to send StowRS requests for DicomWeb peer. This is appended to the contents of --dicomweb_addr flag."
```
Comment (danielbeaudreau) [inline import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: And here.

File: README.md [hunk 8]
```diff
@@ -50,6 +51,8 @@ And command to create configmap from it:
 kubectl create configmap aet-dictionary --from-file=AETs.json
 ```
 
+The AET dictionary JSON can also be specified directly via the "--aet_dictionary_inline" parameter.
+
 Note that any C-FIND query on the ModalitiesInStudy tag will result in 1 QIDO-RS query per modality.
```
Comment (danielbeaudreau) [inline README.md]: Can you move this note to above the "Available AET destinations" section? Now that this section is a bit long, it makes more sense for it to be near where we describe the supported services.

File: README.md (hunk 1 above)
Comment (danielbeaudreau) [reply README.md]: Agreed, can you please deprecate these as well (the peer flags) in a follow-up PR? Thanks

Comment (red1408) [reply README.md]: Ok, in a follow-up PR.

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java (hunk 6 above)
Comment (red1408) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: Ok

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java (hunk 7 above)
Comment (red1408) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java]: Ok

File: README.md (hunk 8 above)
Comment (red1408) [reply README.md]: Ok

**Comment by danielbeaudreau** [comment]
Please resolve conflicts and then I will merge this. Thanks

File: integration_test/run_adapter_and_wait.sh [hunk 9]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (danielbeaudreau) [inline integration_test/run_adapter_and_wait.sh]: I think this file should be deleted, it's a duplicate

Comment (red1408) [reply integration_test/run_adapter_and_wait.sh]: Right, deleted it.

**Pull request merged**